sonar-db/src/main/java/org/sonar/db/permission/GroupPermissionDao.java

```diff
@@ -77,6 +91,28 @@ public void groupsCountByComponentIdAndPermission(DbSession dbSession, List<Long
       });
   }
 
+  /**
+   * @return the permissions granted to the requested group, optionally on the requested project. An
+   * empty list is returned if the group or project do not exist.
+   */
+  public List<String> selectGroupPermissions(DbSession session, long groupId, @Nullable Long projectId) {
+    return session.getMapper(GroupPermissionMapper.class).selectGroupPermissions(groupId, projectId);
+  }
+
+  /**
+   * @return the permissions granted to Anyone virtual group, optionally on the requested project. An
+   * empty list is returned if the project does not exist.
+   * @deprecated not compatible with organizations if {@code projectId} is null. Should have an organization parameter.
+   */
+  @Deprecated
+  public List<String> selectAnyonePermissions(DbSession session, @Nullable Long projectId) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)